### PR TITLE
Add highlights for the week ending May 13, 2022

### PR DESCRIPTION
**Pull request by marians**
Please add your contributions until May 13, 14:00 CEST.

### CODE REVIEW

File: src/content/changes/highlights/2022-05-13.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date: 2022-05-06T10:00:00
+title: Highlights for the week ending May 13, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2022-05-13.md:11]:
```suggestion
- [linkerd2-app](https://github.com/giantswarm/linkerd2-app) version [v0.7.0](https://github.com/giantswarm/linkerd2-app/blob/master/CHANGELOG.md#070---2022-05-13) updates to upstream version [stable-2.11.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.11.2). Before upgrading to this release, please upgrade linkerd2-cni-app to v0.7.0
- [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [v0.7.0](https://github.com/giantswarm/linkerd2-cni-app/blob/master/CHANGELOG.md#070---2022-05-12) updates to upstream version [stable-2.11.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.11.2). This release should be used together with linkerd2-app v0.7.0
```

Comment (Gacko) [reply src/content/changes/highlights/2022-05-13.md:11]:
```suggestion
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [v2.12.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2120---2022-05-13) drops support for `cluster.profile` and reduces default resource requests.
- [external-dns-app](https://github.com/giantswarm/external-dns-app) version [v2.11.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2110---2022-05-10) adds support for `DNSEndpoint` custom resources.
```

**Comment by gianfranco-l** [comment]
nothing from honeybadger

File: src/content/changes/highlights/2022-05-13.md (hunk 1 above)
Comment (ubergesundheit) [reply src/content/changes/highlights/2022-05-13.md:11]: @marians can you also apply the suggestion from marco please?

**Comment by marians** [comment]
@Gacko @ubergesundheit Please check whether your additions are complete now. I think on the first attempt, GitHub has swallowed something.

**Review by ubergesundheit — APPROVED** [review]

**Pull request merged**